examples/config-gcc/config-gcc.ncl

```diff
@@ -51,7 +51,7 @@ let OptLevel = fun label value =>
     contracts.blame label in
 
 let Contract = {
-  pathLibC | doc "Path to libc."
+  path_libc | doc "Path to libc."
```

Comment:
Should we apply the change to configuration option? In real life, their name is dictated by the consuming tool (say, Terraform), most commonly in camelCase than in snake_case.